### DISCUSSION TITLE
Reproduce example from Hugging Face Img2Img

### DISCUSSION THREAD

**Discussion by katarzynasornat** [huggingface/diffusers#6924] [Q&A] [answered]
Hi all! 

I try to reproduce some examples from [here](https://huggingface.co/docs/diffusers/using-diffusers/img2img?fbclid=IwAR1yhqwpgXvk9CMLEWCIjRnZlOXyetDV0HIWByEqb4qkquANHouNZyNokUQ) in Google Colab.

I have a problem with mirroring "jungle", I think I did all the same, running the same code. What I got for stable diffusion 1.5 is below (the right image). What I could do wrong?

![image](https://github.com/huggingface/diffusers/assets/9012956/66f18e15-929c-412e-8358-0d7d5f9cf879)

**Comment by tolgacangoz** [comment]
Hi @katarzynasornat! Here is [my try](https://colab.research.google.com/gist/standardAI/8cee045c82bdcdeee23ef49d0568a7cb/jungle.ipynb).

**Reply by katarzynasornat** [reply]
@standardAI thanks for your answer! I saw the results and still I think that official one looks somehow better. Should I bother myself with this or should I assume is all OK?

**Reply by katarzynasornat** [reply]
I also tried for SDXL and there I am getting the same bad results whereas in the doc they obtain very nice jungle in the background with strenght = 0.5... :(

**Comment by sayakpaul** [comment]
Cc: @stevhliu

**Comment by asomoza** [comment, marked as answer]
It's really hard to reproduce the same exact image with a random seed, also probably the images were made with one of the first versions of diffusers combined with the environment of the user it would drastically affects the results.

Still, you need to play with the parameters to get good results, the guide is just an example, don't be afraid to change things and experiment, this is my try:

![image](https://github.com/huggingface/diffusers/assets/5442875/907e3d91-1a78-4321-88dd-14c7dc4c15d1)

colab: https://colab.research.google.com/drive/1dPOB-DzGuY9sudHSPpE3FeAfsoo0PQ7k?usp=sharing

```python
image = pipeline(prompt, image=init_image, guidance_scale=8.0, num_inference_steps=30, strength=0.6).images[0]
```

This is what you need to know for being able to generate good results with image to image:

> strength is one of the most important parameters to consider and it’ll have a huge impact on your generated image. It determines how much the generated image resembles the initial image. In other words:

> The strength and num_inference_steps parameters are related because strength determines the number of noise steps to add. For example, if the num_inference_steps is 50 and strength is 0.8, then this means adding 40 (50 * 0.8) steps of noise to the initial image and then denoising for 40 steps to get the newly generated image.

> The guidance_scale parameter is used to control how closely aligned the generated image and text prompt are. A higher guidance_scale value means your generated image is more aligned with the prompt, while a lower guidance_scale value means your generated image has more space to deviate from the prompt.
